nexus/celery.py

```diff
@@ -6,11 +6,15 @@
 from celery import Celery
 from django.conf import settings
 from dotenv import load_dotenv
-from raven.contrib.celery import register_signal, register_logger_signal
+from raven.contrib.celery import register_logger_signal, register_signal
 
 # Set the default Django settings module for the 'celery' program.
 load_dotenv(os.path.join(os.path.dirname(os.path.dirname(__file__)), '.env'))
 os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'settings.development')
+# Time period in minutes before next celery beat
+CELERY_BEAT_MINUTES = {
+    'publish_posts_task': 30,
```

Comment:
Let's make it in seconds as discussed in the previous comment.

Also, please pick this up from settings.